superset-frontend/src/dashboard/components/RefreshIntervalModal.tsx

```diff
@@ -104,7 +112,7 @@ class RefreshIntervalModal extends React.PureComponent {
             <FormLabel>{t('Refresh frequency')}</FormLabel>
             <Select
               options={options}
-              value={this.state.refreshFrequency}
+              value={options.find(({ value }) => value === refreshFrequency)}
```

Comment:
can you include a test plan showing that everything still works as intended after this change?

Perhaps it might be better to construct an option object here instead of finding it from the options array, but I don't have a strong feeling one way or another.

Reply:
`<Select
  options={options}
  value={{ value: refreshFrequency }}
  onChange={this.handleFrequencyChange}
/>`
Changed value @etr2460